homeassistant/core.py

```diff
@@ -132,7 +132,12 @@ class HomeAssistant(object):
 
     def __init__(self, loop=None):
         """Initialize new Home Assistant object."""
-        self.loop = loop or asyncio.get_event_loop()
+        if sys.platform == "win32" and loop is None:
+            self.loop = asyncio.ProactorEventLoop()
+            asyncio.set_event_loop(self.loop)
```

Comment:
We shouldn't have to set the event loop here, all things should grab the event loop from hass